modules/c/pages/gs-install.adoc

```diff
@@ -335,6 +445,62 @@ In this instance the release comprises a _ready to adapt_ application project.
 
 Once the install is finished, you can build and run this skeleton app.
 
+=== Install Vector Search for Android
```

Comment:
This is complicated. 

1. 'com.couchbase.lite:couchbase-lite-android-ee:{version-maintenance-android}'` This is CBL-Android, not CBL-C. CBL-C provides native shared library that developers can use to write JNI codes for their application. So CBL-C developers for Android platform will not use `couchbase-lite-android-ee` library.

2. `com.couchbase.lite:couchbase-lite-java-vector-search-arm64` is pure Java so it's not relevant here.

3. `couchbase-lite-android-vector-search-arm64 / x86_64` : This is interesting as users can just use this instead of the native android libraries provided by CBL-C. However, CBL-C 3.2.0 beta is not compatible with `couchbase-lite-java-android-search-arm64/x86_64` beta-2 so the developers cannot use the current `couchbase-lite-java-android-search`. So it would be best that we don't include this instruction here to avoid confusion. But this would work for GA.

4. CBL-C Vector Search provides a downloable file that only contains the vector search libraries that the developer will need to include them in their app. The steps to include the prebuilt native library can be found here: https://developer.android.com/studio/projects/gradle-external-native-builds.

If this is not clear, we can sync up about this tomorrow.